src/test/java/org/folio/rest/api/PatronNoticePoliciesApiTest.java

```diff
@@ -137,6 +138,38 @@ public void canGetAllPatronNoticePolicies() throws MalformedURLException, Interr
     MatcherAssert.assertThat(names, arrayContainingInAnyOrder(firstPolicy.getName(), secondPolicy.getName()));
   }
 
+  @Test
+  public void canGetPatronNoticePolicy() throws MalformedURLException, InterruptedException, ExecutionException,
+    TimeoutException {
+
+    String id = createPatronNoticePolicy(firstPolicy).getJson().getString("id");
+
+    CompletableFuture<JsonResponse> getCompleted = new CompletableFuture<>();
+
+    client.get(patronNoticePoliciesStorageUrl("/" + id), StorageTestSuite.TENANT_ID, ResponseHandler.json(getCompleted));
+
+    JsonResponse response = getCompleted.get(5, TimeUnit.SECONDS);
+
+    MatcherAssert.assertThat(response.getStatusCode(), CoreMatchers.is(200));
```

Comment:
I think that the readability of these asserts could be improved with static imports of the assertion and matcher methods.
```suggestion
    assertThat(response.getStatusCode(), is(200));
```

Reply:
Good idea. Will do it. Thanks.

Reply:
Thanks